runtimes/meta_quest_2_mobile.json.license

```diff
@@ -0,0 +1,2 @@
+Copyright 2022, The Khronos Group Inc.
```

Comment:
:thinking:

Reply:
Assuming you're pointing to 2022? Updated to 2024.